frontend/services/entities/hosts.ts

```diff
@@ -116,6 +118,27 @@ export interface IActionByFilter {
   query: string;
   status: string;
   labelId?: number;
+  currentTeam?: number | null;
+  policyId?: number | null;
+  policyResponse?: PolicyResponse;
+  softwareId?: number | null;
+  softwareTitleId?: number | null;
+  softwareVersionId?: number | null;
+  osName?: string;
+  osVersion?: string;
+  osVersionId?: number | null;
+  // DisableFailingPolicies
```

Comment:
comments

Reply:
are these intentionally here for future addition? If so maybe just communicate that in a comment above or something please.